src/platform/site-wide/side-nav/components/NavItem.js

```diff
@@ -0,0 +1,98 @@
+// Dependencies
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { get } from 'lodash';
+// Relative
+import DuplicateLineLabel from './DuplicateLineLabel';
+import ExpandCollapseButton from './ExpandCollapseButton';
+import LabelText from './LabelText';
+
+const NavItem = ({
+  depth,
+  item,
+  index,
+  renderChildItems,
+  sortedNavItems,
+  toggleItemExpanded,
+}) => {
+  // Derive the item properties.
+  const expanded = get(item, 'expanded');
+  const id = get(item, 'id');
+  const isSelected = get(item, 'isSelected');
+  const label = get(item, 'label', '');
+
+  // Derive the depth booleans.
+  const isFirstLevel = depth === 1;
+  const isDeeperThanSecondLevel = depth >= 2;
+
+  // Caclculate the indentation for the child items.
+  const indentation = isDeeperThanSecondLevel ? 20 * (depth - 1) : 20;
```

Comment:
It would be cool if instead of applying these as just inline style, we could map them into the padding-left design system utilities. Is that possible?

Reply:
Oh! I actually think this loc is not accurate but I've never had links that are 4th level so far to test with 🤔

Reply:
No, no paths should exceed the 3rd level in Pittsburgh Health Care 🙂